README.md

```diff
@@ -5,53 +5,64 @@ a specified C/C++ function.
 It also provides support for versioning of the compiled functions.
 
 This software is distributed under the LGPLv3 license.
-See LICENSE.txt and LICENSE.LESSER.txt for the full text of the license.
+See [LICENSE.txt](LICENSE.txt) and [LICENSE.LESSER.txt](LICENSE.LESSER.txt) for full license details.
 
 An introductory video is available on [YouTube](https://www.youtube.com/watch?v=1p8IajxOgoY).
 
 ## Dependencies
 
-libVersioningCompiler requires:
+To build and use libVersioningCompiler, the following are required:
 
-- Ubuntu 20.04 LTS or greater / MacOS (tested with Sonoma) / Arch Linux or Manjaro / a linux distribution
-- any compiler compliant to the C++17 standard
-- cmake 3.20 or greater
-- zlib
-- libuuid
-- (OPTIONAL) LLVM 18 or greater (tested up to LLVM 19)
+- Operating System:
+  - Ubuntu 20.04 LTS or newer
+  - Arch Linux, Manjaro or Fedora
+  - macOS (tested on Sonoma)
 
-Compiling without the OPTIONAL dependencies will disable some features,
-like the Clang-as-a-library compiler implementation.
+- Build Tools & Libraries:
+  - A C++17-compliant compiler
+  - CMake ≥ 3.20
```

Comment:
can we use the formula "<product> <version_ number> or newer" instead of the greater-or-equal sign, which is not an ASCII symbol (and therefore subject to encoding-related issues)?